pkgs/by-name/co/code-cursor/package.nix

```diff
@@ -54,41 +70,73 @@ let
     runHook postInstall
   '';
 
+  darwinInstallPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications/
+
+    mkdir -p "$out/bin"
+    cat << EOF > "$out/bin/cursor"
+    open -na '$APP_DIR' --args "\$@"
+    EOF
+    chmod +x "$out/bin/cursor"
+    runHook postInstall
+  '';
+
 in
 stdenvNoCC.mkDerivation {
   inherit pname version;
 
-  src = wrappedAppImage;
+  src = if stdenvNoCC.isLinux then wrappedAppImage else source;
```

Comment:
```suggestion
  src = if buildPlatform.isLinux then wrappedAppImage else source;
```

Reply:
The build fails on Darwin with "no build phase specified". We need to tell it we don't need one.